documentation/src/docs/controller-service.md

```diff
@@ -12,6 +12,7 @@ You may obtain a copy of the License at
 *  [Introduction](#introduction)
 *  [Architecture](#architecture)
     - [Stream Management](#stream-management)
+    - [Key Value Tables Management](#keyValue-management)
```

Comment:
Now Pravega also does "ReaderGroup" management. Perhaps that too should be added here.

Reply:
could you please share the write up for it. thanks

Reply:
Here is what I could think of, for ReaderGroups:
```
Reader Group Management
--------------------------------
A Pravega Reader Group is a group of readers that collectively read events from a Stream. 
A single segment in the Stream is assigned to exactly one reader in the ReaderGroup.
A Reader Group has a configuration that decides its checkpointing related behaviour and also what Streams it reads from.
Starting Pravega 0.9, Reader Groups as entities are created on client but also stored on Controller.
This helps facilitate Consumption Based Retention on Streams.
```